src/main/java/be/syntra/devshop/DevshopFront/services/CartServiceImpl.java

```diff
@@ -111,7 +136,7 @@ private CartDto wrap(CartDto currentCart) {
                 .paidCart(currentCart.isPaidCart())
                 .cartCreationDateTime(currentCart.getCartCreationDateTime())
                 .user(currentCart.getUser())
-                .products(currentCart.getProducts())
+                .cartProductDtoList(currentCart.getCartProductDtoList())
                 .build();
     }
 
```

Comment:
take a look at the setCartToFinalized method. I know it's legacy code now, but it makes no sense. The Cart is set as paid == false. BUT THE METHOD IS CALLED WITHIN THE METHOD TO PAY!!! #tired

Reply:
i removed the helpermethod that wasn't helping, setting the cart to finalized is now done in the method that makes the call to the BE.  And i removed setting the paid property to false because we don't do anything payment-related at the moment and its false by default, the idea is that when something payment related is implemented the property can be manipulated in the new code and that it isn't changed here.